packages/cli-e2e/__tests__/angular.specs.ts

```diff
@@ -14,6 +20,7 @@ import {isSearchRequest} from '../utils/platform';
 import {EOL} from 'os';
 import stripAnsi from 'strip-ansi';
 import {ProcessManager} from '../utils/processManager';
+import {Runtime} from 'node:inspector';
```

Comment:
```suggestion
import type {Runtime} from 'inspector';
```

Reply:
we still need the `node:` prefix otherwise we get a eslint warning
![image](https://user-images.githubusercontent.com/12199712/117067146-a4fee580-acf7-11eb-8b2f-78acb5dd6a21.png).

We can either use `node:` or disable the eslint rule